airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3778,19 +3778,47 @@ components:
           type: string
     OperatorWebhook:
       type: object
-      required:
-        - executionUrl
       properties:
-        executionUrl:
-          type: string
-          description: The URL to call to execute the webhook operation via POST request.
-        executionBody:
-          type: string
-          description: If populated, this will be sent with the POST request.
         webhookConfigId:
           type: string
           format: uuid
           description: The id of the webhook configs to use from the workspace.
+        webhookType:
```

Comment:
This is a nice little affordance, lets the frontend use `if (operatorWebhook.webhookType === "dbtCloud")` to narrow the type checking to dbtCloud-specific fields.